lib/actions/model.js

```diff
@@ -61,4 +61,26 @@ module.exports = {
       return xhr;
     };
   },
+
+  validate(brainstemKey, modelId, attributes, options = {}) {
+    const {
+      validateOptions = {},
+      preValidateAction,
+      postValidateAction,
+      trackKey,
+    } = options;
+
+    const storageManager = StorageManager.get();
+
+    return (dispatch) => {
+      if (preValidateAction) dispatch(preValidateAction);
+
+      const Model = storageManager.storage(brainstemKey).model;
+      const validationErrors = new Model({ id: modelId }).validate(attributes, validateOptions);
```

Comment:
No. Brainstem does return the same instance of the model. That's a feature:
singleton models.